mesonbuild/cargo/interpreter.py

```diff
@@ -269,6 +272,9 @@ class Manifest:
     subdir: str
     path: str = ''
 
+    def __post_init__(self) -> None:
+        self.features.setdefault('default', [])
```

Comment:
`self.features` is a `List[str]`, it doesn't have a `setdefault` method. IS this in the wrong commit?

Reply:
It is a `T.Dict[str, T.List[str]]`